FirstNLastPos.java

```diff
@@ -0,0 +1,53 @@
+//TC: O(Log (n*k))
```

Comment:
can you explain your variables , what is k here?

Reply:
N would be the length of the input array to find the first element & 
K would be the length of the array from the first element index to the last.